lib/op.ml

```diff
@@ -120,6 +121,7 @@ type op1 =
 
 let pp_op1 hexa o = match o with
 | Not -> "!"
+| LogicalNot -> "~"
```

Comment:
Isn't "LogicalNot" the same operation as "Inv" below?

Reply:
Thanks for pointing this out -- seems that the `Inv` operator was added after we started working on this, and I hadn't noticed. I've removed `LogicalNot` and switched to using `Inv`.